client/galaxy/scripts/mvc/ui/ui-slider.js

```diff
@@ -26,52 +24,25 @@ var View = Backbone.View.extend({
         this.$slider = this.$(".ui-form-slider-element");
 
         // add text field event
-        var pressed = [];
         this.$text
-            .on("change", function() {
-                self.value($(this).val());
+            .on("change", e => {
+                this.value(e.currentTarget.value);
             })
-            .on("keyup", e => {
-                pressed[e.which] = false;
-            })
-            .on("keydown", function(e) {
-                var v = e.which;
-                pressed[v] = true;
-                if (self.model.get("is_workflow") && pressed[16] && v == 52) {
```

Comment:
I think what this logic was trying to do was to allow `${workflow_parameter}` entries when this widget is used in a workflow context, and it looks like the input replacement below might restrict this.

Reply:
I'll do some more testing to make sure this works in a workflow context. I'm not super familiar with how the workflow works but my though process was that this if statement would cover the use case you described
```
if (this._isParameter(input.value)) {
            return;
}
```

Reply:
@WilliamHolden Yep, that handles it, just tested it.  Thanks for checking, and simplifying the logic here!